python/psautohint/fdTools.py

```diff
@@ -195,16 +199,16 @@ def buildBlueLists(self):
                 zoneBuffer = 2 * self.BlueFuzz + 1
                 for pair in bluePairList[1:]:
                     if prevPair[0] > pair[1]:
-                        print("Error in FDDict %s. The top of zone %s at %s "
-                              "overlaps zone %s with the bottom at %s." %
-                              (self.DictName, prevPair[2], prevPair[0],
-                               pair[2], pair[1]))
+                        log.error("In FDDict %s. The top of zone %s at %s "
+                                  "overlaps zone %s with the bottom at %s.",
+                                  self.DictName, prevPair[2], prevPair[0],
+                                  pair[2], pair[1])
                     elif abs(pair[1] - prevPair[0]) <= zoneBuffer:
-                        print("Error in FDDict %s. The top of zone %s at %s "
-                              "is within the min spearation limit (%s units) "
-                              "of zone %s with the bottom at %s." %
-                              (self.DictName, prevPair[2], prevPair[0],
-                               zoneBuffer, pair[2], pair[1]))
+                        log.error("In FDDict %s. The top of zone %s at %s is "
+                                  "within the min spearation limit (%s units) "
+                                  "of zone %s with the bottom at %s.",
+                                  self.DictName, prevPair[2], prevPair[0],
+                                  zoneBuffer, pair[2], pair[1])
```

Comment:
all these "errors".. it doesn't look like we are handling them in any sort of way, besides printing the message.
it they are something that the library can continue with maybe their severity is "warning"? Just wondering really.

Reply:
Good point, I just followed “Error” in the old blindly, but warnings might make more sense indeed.